unit/Makefile

```diff
@@ -16,14 +17,14 @@ LIBS += ../src/ansi-c/ansi-c$(LIBEXT) \
         ../src/cpp/cpp$(LIBEXT) \
         ../src/json/json$(LIBEXT) \
         ../src/linking/linking$(LIBEXT) \
-        ../src/util/util$(LIBEXT) \
-        ../src/big-int/big-int$(LIBEXT) \
-        ../src/goto-programs/goto-programs$(LIBEXT) \
         ../src/pointer-analysis/pointer-analysis$(LIBEXT) \
         ../src/langapi/langapi$(LIBEXT) \
         ../src/assembler/assembler$(LIBEXT) \
         ../src/analyses/analyses$(LIBEXT) \
+        ../src/goto-programs/goto-programs$(LIBEXT) \
         ../src/solvers/solvers$(LIBEXT) \
+        ../src/util/util$(LIBEXT) \
+        ../src/big-int/big-int$(LIBEXT) \
```

Comment:
Ahha bingo! I'll apply this modification to #1111 and once that's in I'll rebase this on top of it.